doc/man3/OSSL_PARAM_allocate_from_text.pod

```diff
@@ -28,28 +28,21 @@ size (see L<OSSL_PARAM(3)> for more information).
 
 OSSL_PARAM_allocate_from_text() takes a control I<key>, I<value> and
 value size I<value_n>, and given a parameter descriptor array
-I<paramdefs>, it converts the value to something suitable for
-L<OSSL_PARAM(3)> and stores that in the buffer I<buf>, and modifies
-the parameter I<to> to match.
-I<buf_n>, if not NULL, will be assigned the number of bytes used in
-I<buf>.
-If I<buf> is NULL, only I<buf_n> will be modified, everything else is
-left untouched, allowing a caller to find out how large the buffer
-should be.
-I<buf> needs to be correctly aligned for the type of the B<OSSL_PARAM>
-I<key>.
-If <found> is not NULL, it is set to 1 if the parameter can be located and
-to 0 otherwise.
-
-The caller must remember to free the data of I<to> when it's not
-useful any more.
+I<paramdefs>, it converts the value to something suitable for an
```

Comment:
Maybe it should say something about looking up the value where key matching the key from paramdefs?

Reply:
Looking up the value?  I don't understand...

Reply:
I assume that in paramdefs, it's going to look for an OSSL_PARAM who's key == the key from the parameters

Reply:
Yes. And it then uses that item's data type to figure out what to do with *value*. Is that what you meant?

Reply:
Yes. And it's probably very obvious to you. But things like that should be documented.

Reply:
Ok, I've rephrased that whole paragraph.  Does it seem better?

Reply:
(and yeah, this *is* hard...  as you say, it's obvious to me, so it's hard to see what everyone else don't necessarily see.  So thank you for insisting)